pages.fr/common/csh.md/csh.md

```diff
@@ -0,0 +1,21 @@
+# csh
+
+> Le shell (interprèteur de commandes) affiche une syntaxe proche de C.
+> Voir également : tcsh.
+> Plus d'informations : https://www.mkssoftware.com/docs/man1/csh.1.asp.
+
+- Démarrer une session interactive:
+
+`csh`
+
+- Démarrer une session interactive sans prendre en compte le fichier de configuration au démarrage:
+
+`csh -f`
+
+- Exécuter une commande:
+
+`csh -c "{{echo 'Exécution de la commande echo par csh'}}"`
+
+- Exécuter un script:
+
+`csh {{chemin/d_acces_au/script.csh}}`
```

Comment:
```suggestion
`csh {{chemin/vers/script.csh}}`
```

Referring the structure at 

https://github.com/tldr-pages/tldr/blob/99ecf782b809f72e55c434105c53dceb372f4eb7/contributing-guides/translation-templates/common-arguments.md?plain=1#L18